### PR TITLE
Feelnc tool

### PR DESCRIPTION
**Pull request by stephanierobin**
Hi, Here you can find a new tool for annotation of long non coding RNAs.

### CODE REVIEW

File: tools/feelnc/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Sequence Analysis
+description: Galaxy wrapper for FEELnc version 0.1.1
```
Comment (bgruening) [inline tools/feelnc/.shed.yml]: you can remove the version information here

File: tools/feelnc/feelnc_wrapper.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="feelnc" name="FEELnc" profile="17.01" version="0.1.1">
+    <description>FlExible Extraction of LncRNA</description>
+    <requirements>
+        <requirement type="package" version="0.1.1">feelnc</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+FEELnc_pipeline.sh 
+--candidate=${transcripts}
```
Comment (bgruening) [inline tools/feelnc/feelnc_wrapper.xml]: data and text parameters should be single quoted. See here for more information: http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=quote#command-formatting

File: tools/feelnc/feelnc_wrapper.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <outputs>
+        <data format="gtf" name="candidate_lncRNA" from_work_dir="out_feelnc/codpot/candidate_lncRNA.codpot.lncRNA.gtf" label="lncRNA annotation with ${tool.name} on ${on_string}" />
+        <data format="gtf" name="candidate_mRNA" from_work_dir="out_feelnc/codpot/candidate_lncRNA.codpot.mRNA.gtf" label="mRNA annotation with ${tool.name} on ${on_string}" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="transcripts" value="transcript_chr38.gtf" />
+            <param name="annotation" value="annotation_chr38.gtf" />
+            <param name="genome" value="genome_chr38.fa" />
+            <output name="candidate_lncRNA" file="candidate_lncRNA.gtf.lncRNA.gtf" compare="sim_size" />
```
Comment (bgruening) [inline tools/feelnc/feelnc_wrapper.xml]: Why do you use sim_size here, are the outputs randomly sorted?

File: tools/feelnc/.shed.yml (hunk 1 above)
Comment (stephanierobin) [reply tools/feelnc/.shed.yml]: done

File: tools/feelnc/feelnc_wrapper.xml (hunk 2 above)
Comment (stephanierobin) [reply tools/feelnc/feelnc_wrapper.xml]: done

File: tools/feelnc/feelnc_wrapper.xml (hunk 3 above)
Comment (stephanierobin) [reply tools/feelnc/feelnc_wrapper.xml]: @bgruening  thanks for this review
yes the outputs of feelNC are randomly sorted

Comment (bgruening) [reply tools/feelnc/feelnc_wrapper.xml]: Could you use the `sort` option from here? https://docs.galaxyproject.org/en/latest/dev/schema.html#id84

Comment (cmonjeau) [reply tools/feelnc/feelnc_wrapper.xml]: It's good with the sort option. Thanks @bgruening

**Comment by bgruening** [comment]
Cool thanks @stephanierobin and @cmonjeau!

**Pull request merged**